pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
+
+  src = fetchgit {
```

Comment:
You're right, I had a typo in the author's name and I jumped to conclusions. Fixing!